RestaurantsNearMe/Presentation/SplashScreenViewModel.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  SplashScreenViewModel.swift
+//  RestaurantsNearMe
+//
+//  Created by Colin Evans on 2023-11-24.
+//
+
+import Foundation
+import Combine
+import CloudKit
+
+class SplashScreenViewModel: ObservableObject {
+  @Published var isFetching = false
+  @Published var fetchingError: String?
```

Comment:
I'm not sure the best way to handle error propagation via async calls.
My first instinct is to catch the error via the fetch and mapping the result to a published `var`